cfme/base/login.py

```diff
@@ -1,5 +1,7 @@
+from cfme.exceptions import CFMEException
```

Comment:
Required: move it down as this intermal import from cfme.exceptions import CFMEException:

from widgetastic.exceptions import NoSuchElementException
from widgetastic.widget import View
from widgetastic_manageiq import SettingsNavDropdown
from widgetastic_patternfly import NavDropdown, VerticalNavigation, FlashMessages	 

from cfme.exceptions import CFMEException

Reply:
Updated